tracer/src/Datadog.Trace/AppSec/BlockException.cs

```diff
@@ -29,17 +29,20 @@ internal BlockException(System.Runtime.Serialization.SerializationInfo info, Sys
         {
         }
 
-        public BlockException(IResult result, bool reported = false)
+        public BlockException(string triggerData, ulong aggregatedTotalRuntime, ulong aggregatedTotalRuntimeWithBindings, bool reported = false)
         {
-            Result = result;
+            TriggerData = triggerData;
+            AggregatedTotalRuntime = aggregatedTotalRuntime;
+            AggregatedTotalRuntimeWithBindings = aggregatedTotalRuntimeWithBindings;
             Reported = reported;
         }
 
-        internal IResult Result { get; }
+        public string TriggerData { get; }
 
-        public bool Reported { get; }
+        public ulong AggregatedTotalRuntime { get; }
+
+        public ulong AggregatedTotalRuntimeWithBindings { get; }
 
-        // can give a significant performance boost, this exception is currently caught and logged by the host web server
-        public override string ToString() => "BlockException";
```

Comment:
we're aiming not leak this exception on the hosting web server any more, we don't leak this exception any more, so I think it's okay to remove this now.